pep/app/utils/search.ts

```diff
@@ -262,13 +262,18 @@ export function groupCountsByRange(
  */
 export function copySearchToController(toController: Controller & SearchController) {
     const searchController = getOwner(toController).lookup(`controller:search`);
+    const config: ConfigurationService = getOwner(toController).lookup('service:configuration');
+    const defaultFields = config.base.search.terms.defaultFields;
```

Comment:
will this pull in the user's previously saved search term field preferences to use instead of the admin config defaults, when loading/reloading the app on the homepage directly?

Reply:
Yes - it gets it from the search controller (which does that part)

Reply:
I'm only seeing that happen if you load the app on the /search route directly on your branch, if i load the app directly on the homepage, it just renders the admin config's default fields. Which I think is because the search controller only gets the user prefs' term fields when its `setupController()` is run, which only happens if you actually enter that route.